datafusion/core/tests/parquet/arrow_statistics.rs

```diff
@@ -0,0 +1,270 @@
+    assert_eq!(arrow_stats.min().len(), 1, "expected 1 row group");
+
+    // min is -1
+    let expect = Int64Array::from(vec![-1]);
+    let expect = Arc::new(expect) as ArrayRef;
+    assert_eq!(min, &expect);
+
+    // max is 4
+    let expect = Int64Array::from(vec![4]);
+    let expect = Arc::new(expect) as ArrayRef;
+    assert_eq!(arrow_stats.max(), &expect);
+
+    // 2 nulls
+    let expect = UInt64Array::from(vec![2]);
+    let expect = Arc::new(expect) as ArrayRef;
+    assert_eq!(arrow_stats.null_count(), &expect);
+
+    // 8 rows
+    let expect = UInt64Array::from(vec![8]);
+    let expect = Arc::new(expect) as ArrayRef;
+    assert_eq!(arrow_stats.row_count(), &expect);
+}
+
+#[tokio::test]
+async fn test_two_row_group_with_null() {
+    let row_per_group = 10;
+    let reader = parquet_file(2, 4, 17, row_per_group);
+
+    let statistics =
+        arrow_statistics::ParquetColumnStatistics::from_parquet_statistics(&reader)
+            .expect("getting statistics");
+
+    // only 1 column
+    assert_eq!(statistics.len(), 1, "expected 1 column");
+
+    let arrow_stats = parquet_stats_to_arrow(&DataType::Int64, &statistics[0]).unwrap();
+    println!("Arrow statistics:\n {arrow_stats:#?}");
+
+    // 2 row groups
+    let mins = arrow_stats.min();
+    assert_eq!(mins.len(), 2, "expected 2 row groups");
+
+    // mins are [4, 14]
+    let expect = Int64Array::from(vec![4, 14]);
+    let expect = Arc::new(expect) as ArrayRef;
+    assert_eq!(mins, &expect);
+
+    // maxs are [13, 16]
+    let maxs = arrow_stats.max();
+    let expect = Int64Array::from(vec![13, 16]);
+    let expect = Arc::new(expect) as ArrayRef;
+    assert_eq!(maxs, &expect);
+
+    // nuls are [0, 2]
+    let nulls = arrow_stats.null_count();
+    let expect = UInt64Array::from(vec![0, 2]);
+    let expect = Arc::new(expect) as ArrayRef;
+    assert_eq!(nulls, &expect);
+
+    // row counts are [10, 5]
+    let row_counts = arrow_stats.row_count();
+    let expect = UInt64Array::from(vec![10, 5]);
+    let expect = Arc::new(expect) as ArrayRef;
+    assert_eq!(row_counts, &expect);
+}
+
+#[tokio::test]
+async fn test_two_row_groups_with_all_nulls_in_one() {
+    let row_per_group = 5;
+    let reader = parquet_file(4, -2, 2, row_per_group);
+
+    let statistics =
+        arrow_statistics::ParquetColumnStatistics::from_parquet_statistics(&reader)
+            .expect("getting statistics");
+
+    // only 1 column
+    assert_eq!(statistics.len(), 1, "expected 1 column");
+
+    let arrow_stats = parquet_stats_to_arrow(&DataType::Int64, &statistics[0]).unwrap();
+    println!("Arrow statistics:\n {arrow_stats:#?}");
+
+    // 2 row groups
+    let mins = arrow_stats.min();
+    assert_eq!(mins.len(), 2, "expected 2 row groups");
+
+    // mins are [-2, null]
+    assert!(mins.is_null(1));
+    // check the first value -2
+    // let expect = Int64Array::from(vec![-2]);
+    // let expect = Arc::new(expect) as ArrayRef;
+    // assert_eq!(mins, &expect);
```

Comment:
It is getting late and I am tired. I will need to figure out how to get the first value to verify it